sequencing/ATACSeq/preprocessing/fragmentDistribution.r

```diff
@@ -28,32 +41,61 @@ calcNucleoProps<-function(fraglen.stand){
 	return(c(freeSum, monoSum, diSum, triSum, otherSum))
 }
 
+fdsPlots <- function(listSamples){
+  counter = 1
+  list_plots <- vector("list", 10)
+  samples <- ls(listSamples)
+  for(i in 1:length(listSamples)) {
+    df <-data.frame(listSamples[i])
+    colnames(df)<- c("Var1", "Freq")
+    df$sample <-samples[i]
+    list_plots[[i]] <- df
+    counter = counter + 1
+  }
+  bigdf <-ldply(list_plots, data.frame)
+  bigdf[,1]<- as.integer(as.character(bigdf[,1]))
```

Comment:
Why are you converting to a character then to a integer. Also, what is this column representing. If at all possible, could you use the name of the column instead of its index [,1]?
If not, a comment would be nice.